synthesizer/event.h

```diff
@@ -244,6 +245,8 @@ class NPlayEvent : public PlayEvent {
       NPlayEvent(BeatType beatType);
       const Note* note() const       { return _note; }
       void setNote(const Note* v)    { _note = v; }
+      int originatingStaff() const   { return _origin; }
+      void originatingStaff(int i)   { _origin = i; }
```

Comment:
This is a setter. Please use setOriginatingStaff. Do we really need it ? We can't get it from the stored Note* ?